packages/cxl-lumo-styles/scss/themes/vaadin-context-menu-list-box.scss

```diff
@@ -12,3 +12,25 @@
     box-shadow: inset 0 0 0 ($vaadin-focus-ring-thickness * 0.5) var(--lumo-primary-color);
   }
 }
+
+:host([theme~="cxl-marketing-nav"]) {
+  box-shadow: none !important; /* stylelint-disable-line declaration-no-important */
+  // box-shadow: 0 0 0 1px var(--lumo-shade-5pct), var(--lumo-box-shadow-m);
+}
+
+@media screen and (min-width: 782px) {
+  :host([theme~="cxl-marketing-nav"]) {
+    width: var(--cxl-vaadin-context-menu-item-max-width);
+    padding: 0 10px;
+  }
+
+  :host([theme~="cxl-marketing-nav"].vaadin-menu-list-box--submenu)::before {
+    position: absolute;
+    content: '';
+    top: 16px;
+    left: 0;
+    bottom: 16px;
+    width: 1px;
+    background-color: var(--lumo-shade-20pct);
+  }
```

Comment:
What does it style? Can we use lumo vars instead of absolute numbers or will it fail?